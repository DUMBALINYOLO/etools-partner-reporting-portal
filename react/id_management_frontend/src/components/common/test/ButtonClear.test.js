import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ButtonClear from '../ButtonClear';

describe('Clear button component', () => {
    it('renders clear button', () => {
        const onClick = jest.fn();

        const wrapper = shallow(<ButtonClear
            onClick={onClick}
        />);

        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});