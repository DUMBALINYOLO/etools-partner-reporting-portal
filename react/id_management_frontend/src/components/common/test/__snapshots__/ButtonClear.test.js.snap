// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clear button component renders clear button and calls click 1`] = `
<WithStyles(Button)
  color="primary"
  onClick={[Function]}
>
  Clear
</WithStyles(Button)>
`;
